homeassistant/helpers/device_registry.py

```diff
@@ -104,6 +115,18 @@ def async_get_device(
                 return device
         return None
 
+    @callback
+    def async_get_deleted_device(
```

Comment:
This should be a private function. 
```suggestion
    def _async_get_deleted_device(
```